src/fideslang/models.py

```diff
@@ -239,29 +286,51 @@ class DataUse(FidesModel):
     parent_key: Optional[FidesKey] = None
     legal_basis: Optional[LegalBasisEnum] = Field(
         default=None,
-        description="The legal basis category of which the data use falls under. This field is used as part of the creation of an exportable data map.",
+        description="Deprecated. The legal basis category of which the data use falls under. This field is used as part of the creation of an exportable data map.",
     )
     special_category: Optional[SpecialCategoriesEnum] = Field(
         default=None,
-        description="The special category for processing of which the data use falls under. This field is used as part of the creation of an exportable data map.",
+        description="Deprecated. The special category for processing of which the data use falls under. This field is used as part of the creation of an exportable data map.",
     )
     recipients: Optional[List[str]] = Field(
         default=None,
-        description="An array of recipients when sharing personal data outside of your organization.",
+        description="Deprecated. An array of recipients when sharing personal data outside of your organization.",
     )
-    legitimate_interest: bool = Field(
+    legitimate_interest: Optional[bool] = Field(
         default=False,
-        description="A boolean representation of if the legal basis used is `Legitimate Interest`. Validated at run time and looks for a `legitimate_interest_impact_assessment` to exist if true.",
+        description="Deprecated. A boolean representation of if the legal basis used is `Legitimate Interest`. Validated at run time and looks for a `legitimate_interest_impact_assessment` to exist if true.",
     )
     legitimate_interest_impact_assessment: Optional[AnyUrl] = Field(
         default=None,
-        description="A url pointing to the legitimate interest impact assessment. Required if the legal bases used is legitimate interest.",
+        description="Deprecated. A url pointing to the legitimate interest impact assessment. Required if the legal bases used is legitimate interest.",
     )
+
     is_default: bool = is_default_field
 
     _matching_parent_key: classmethod = matching_parent_key_validator
     _no_self_reference: classmethod = no_self_reference_validator
 
+    @root_validator
+    @classmethod
+    def deprecate_fields(cls, values: Dict) -> Dict:
+        """
+        Warn of Data Use fields pending deprecation.
+        """
+        deprecated_fields = [
+            "legal_basis",
+            "recipients",
+            "special_category",
+            "legitimate_interest",
+            "legitimate_interest_impact_assessment",
+        ]
+        for field in deprecated_fields:
+            if values.get(field) is not None:
```

Comment:
Nit: `None` is the default fallback value for `.get`

```suggestion
            if values.get(field):
```

Reply:
My intent here was to show a warning if anything at all was passed in that wasn't null -

Reply:
Example from one of my tests - if DataUse.legitimate_interest is set to False, if we use `values.get(field)` there is no deprecation warning thrown, but I think there should be.  They've set a value.

Reply:
Makes sense 🤔